src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Excavator(), new Bulldozer(), new Truck()};
```

Comment:
Kind of overloaded expression, can you please create machine instances in separate variables